src/lib/components/CommunitySelectionModal/CommunityListItem.js

```diff
@@ -5,80 +5,44 @@
 // under the terms of the MIT License; see LICENSE file for more details.
 
 import { i18next } from "@translations/i18next";
-import _truncate from "lodash/truncate";
 import _capitalize from "lodash/capitalize";
 import PropTypes from "prop-types";
 import React, { useContext } from "react";
-import { Image } from "react-invenio-forms";
-import { Button, Icon, Item, Label } from "semantic-ui-react";
+import { Button, Icon, Label } from "semantic-ui-react";
+import { CommunityCompactItem } from "../Communities/search";
 import { CommunityContext } from "./CommunityContext";
 
 export const CommunityListItem = ({ result }) => {
   const { setLocalCommunity, getChosenCommunity, userCommunitiesMemberships } =
     useContext(CommunityContext);
 
-  const { metadata, ui } = result;
-  const linkToCommunityPage = result.links.self_html;
-  const linkToLogo = result.links.logo;
+  const { metadata } = result;
   const itemSelected = getChosenCommunity()?.id === result.id;
-  const typeL10n = ui.type?.title_l10n;
-  const isRestricted = result.access.visibility === "restricted";
   const userMembership = userCommunitiesMemberships[result["id"]];
 
-  return (
-    <Item key={result.id} className={itemSelected ? "selected" : ""}>
-      <Image
-        size="tiny"
-        className="community-logo"
-        src={linkToLogo}
-        fallbackSrc="/static/images/square-placeholder.png"
-        as={Item.Image}
-      />
+  const actions = (
+    <Button
+      content={itemSelected ? i18next.t("Selected") : i18next.t("Select")}
+      size="small"
+      positive={itemSelected}
+      onClick={() => setLocalCommunity(result)}
+      aria-label={i18next.t("Select ") + metadata.title}
```

Comment:
I think this translation in not 100% correct: the variable should be inside the translation, because some languages might write the `Select` after the variable, or in another way :)